js/main.js

```diff
@@ -27,6 +38,67 @@ function clickSave(event) {
   data.entries.unshift(entryObj);
   $image.setAttribute('src', 'images/placeholder-image-square.jpg');
   $newEntryForm.reset();
+  data.view = 'entries';
+  location.reload();
```

Comment:
😱 😱 😱 😱

By doing this, you are reloading the entire page, which is extremely resource-intensive. Rather than reload the page, you should be creating a DOM tree for the new entry, and add it to the DOM yourself at the top of your list. That way the entry is visible without refreshing the page.

Reply:
Done.